.ruby-version

```diff
@@ -1 +1 @@
-2.1.5
+2.5.1
```

Comment:
Does the gem itself not need a version bump for this? Same with the bundler version update for the dev dependency?

Reply:
No, these changes only affect gem development & testing -- none of it should affect the gem when it's used by other apps.